ui/app/src/components/Search/utils.ts

```diff
@@ -364,7 +364,7 @@ export const useSearchState = ({ searchParameters, onSelect }: useSearchStatePro
           showPreviewDialog({
             type: 'page',
             title,
-            url: `${guestBase}${getPreviewURLFromPath(item.path)}?crafterCMSGuestDisabled=true`
+            url: `${guestBase}${getPreviewURLFromPath(url)}?crafterCMSGuestDisabled=true`
```

Comment:
Notice on line 350 path is getting renamed to url. Let's fix that and review if there's any other misuse here.